garage/tf/regressors/gaussian_conv_regressor.py

```diff
@@ -86,7 +86,7 @@ def __init__(self,
                             np.zeros(output_dim),
                             np.full(output_dim, np.log(init_std))
                         ],
-                        axis=0)
+                        axis=0)  # yapf: disable
```

Comment:
Is this due to a new version of yapf?

Reply:
I think so but I don't remember.

Reply:
Okay, I just thought it's weird that nothing changed in this file and yapf complained.